apis/swagger.yml

```diff
@@ -518,6 +516,10 @@ paths:
           description: "Return all containers. By default, only running containers are shown"
           type: "boolean"
           default: false
+        - name: "filters"
+          in: "query"
+          description: "Filters in json string(type map[string][]string in Golang). Support filter key [ id label name status ]. More detailed, [ id name status ] support regular expression. [label] support equal and unequal operator."
```

Comment:
Could you change this to make it more readable, like : 
https://github.com/moby/moby/blob/master/api/swagger.yaml#L4325-L4341